lib/core/src/rcGlobal.cpp

```diff
@@ -15,42 +15,39 @@
 
 int ProcessType = CLIENT_PT;
 
-char *dataObjCond[] = {
-    ALL_KW, 		/* operation done on all replica */
-    COPIES_KW, 		/* the number of copies */
-    EXEC_LOCALLY_KW, 	/* execute locally */
-    FORCE_FLAG_KW, 		/* force update */
-    CLI_IN_SVR_FIREWALL_KW, /* client behind same firewall */
-    REG_CHKSUM_KW, 		/* register checksum */
-    VERIFY_CHKSUM_KW, 	/* verify checksum */
-    OBJ_PATH_KW,		/* logical path of the object */
-    RESC_NAME_KW,		/* resource name */
-    DEST_RESC_NAME_KW,	/* destination resource name */
-    BACKUP_RESC_NAME_KW,	/* destination resource name */
-    DATA_TYPE_KW,		/* data type */
-    DATA_SIZE_KW,
-    CHKSUM_KW,
-    VERSION_KW,
-    FILE_PATH_KW,		/* the physical file path */
-    REPL_NUM_KW,		/* replica number */
-    REPL_STATUS_KW, 	/* status of the replica */
-    DATA_INCLUDED_KW,	/* data included in the input bytes buffer */
-    DATA_OWNER_KW,
-    DATA_OWNER_ZONE_KW,
-    DATA_EXPIRY_KW,
-    DATA_COMMENTS_KW,
-    DATA_CREATE_KW,
-    DATA_MODIFY_KW,
-    DATA_ACCESS_KW,
-    DATA_ACCESS_INX_KW,
-    NO_OPEN_FLAG_KW,
-    STREAMING_KW,
-    DATA_ID_KW,
-    COLL_ID_KW,
-    STATUS_STRING_KW,
-    DATA_MAP_ID_KW,
-    "ENDOFLIST"
-};
+char* dataObjCond[] = {ALL_KW, /* operation done on all replica */
+                       COPIES_KW, /* the number of copies */
+                       EXEC_LOCALLY_KW, /* execute locally */
+                       FORCE_FLAG_KW, /* force update */
```

Comment:
Let's keep the original formatting for this and wrap it in `clang-format off/on`.

Reply:
Done.